### ISSUE TITLE
filter_storms - 'dict' object has no attribute 'groupby'

### ISSUE THREAD

**Issue by raybellwaves** [tropycal/tropycal#67]
Thanks for working on this.

I believe there is an issue with [filter_storms](https://tropycal.github.io/tropycal/_modules/tropycal/tracks/dataset.html#TrackDataset.filter_storms) (unless I am using it incorrectly).

```
import tropycal.tracks as tracks

hurdat_atl = tracks.TrackDataset(basin='north_atlantic', source='hurdat')

hurdat_atl.filter_storms()

---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-13-1aeecebfc63f> in <module>
----> 1 hurdat_atl.filter_storms()

~\AppData\Local\Continuum\anaconda3\lib\site-packages\tropycal\tracks\dataset.py in filter_storms(self, year_range, date_range, thresh, subset_domain, doInterp, return_keys)
   2237 
   2238         if return_keys:
-> 2239             return [g[0] for g in points.groupby("stormid")]
   2240         else:
   2241             return p

AttributeError: 'dict' object has no attribute 'groupby'
```

It seems ok with `return_keys=False`

![image](https://user-images.githubusercontent.com/17162724/73955750-d78e1880-48d1-11ea-8dcb-d830772444bc.png)

**Comment by tomerburg** [comment]
Thanks for using Tropycal! This was a bug that was fixed with PR #64 after the release of version 0.1.0; by default, it returns a list of every storm ID that matches the requested criteria (if none is passed, then it returns a list of every storm ID in the TrackDataset object). This will be updated with the next release of Tropycal.

**Comment by obaydar** [comment]
![yenu](https://user-images.githubusercontent.com/54600768/82081659-b7ca7580-96b4-11ea-8aa8-b72f725adf51.png)

I can't get the storms from 2019, I will be appreciated if you can help. Thanks in advance.

**Comment by tomerburg** [comment]
Thank you for pointing this out! This was the result of NHC adding 2019 storms to the HURDATv2 file, while removing past storms from the Best Track directory. This has been resolved with PR #76 and will be implemented in the next release of tropycal v0.2. As such, I am closing this issue.
